themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl

```diff
@@ -62,16 +63,16 @@
               class="thumb js-thumb {if $image.id_image == $product.default_image.id_image} selected js-thumb-selected {/if}"
               data-image-medium-src="{$image.bySize.medium_default.url}"
               data-image-large-src="{$image.bySize.large_default.url}"
-              src="{$image.bySize.home_default.url}"
+              src="{$image.bySize.small_default.url}"
```

Comment:
This means it's not the same image that is being used thus the quality could drop?

Reply:
yes, slightly, it's a small image because it's for the small image below the cover-image